site/_posts/2023-07-11-get-more-value-out-of-fusionauth.md

```diff
@@ -0,0 +1,321 @@
+{
+    //returns HttpClient with correct setting for instance
+    HttpClient httpClient = GetClient();
+
+    //initialize the return value 
+    ReturnValue returnVal = new ReturnValue()
+    {
+        success = false
+    };
+
+    try
+    {
+        //set the endpoint for creating a user
+        string apiURL = $"/api/group/member";
+
+        var payload = FormatGroupUserJSON(groups);
+        var content = new StringContent(payload, Encoding.UTF8, "application/json");
+
+        HttpResponseMessage response = await httpClient.PostAsync($"{Constants.BASE_URL}{apiURL}", content);
+
+        if (response.IsSuccessStatusCode)
+        {
+
+            string responseBody = await response.Content.ReadAsStringAsync();
+            JObject result = JObject.Parse(responseBody);
+            returnVal.success = true;
+            returnVal.result = $"Member ID for added user: {result["members"].First().First()[0]["id"]}";
+        }
+        else
+        {
+            returnVal.result = $"Adding user to the group failed.\nPayload\n{payload}\n{response}";
+        }
+
+    }
+    catch (Exception ex)
+    {
+        Console.WriteLine($"There was an error in the request {ex.Message}");
+        returnVal.result = ex.Message;
+    }
+
+
+    return returnVal;
+}
+```
+
+<br>
+
+## Branding Your User Management with FusionAuth
+*“Design is the silent ambassador of your brand.”*
+
+One of the best parts about using the APIs to allow application admins to manage their own users is they never have to leave your application. Users are accustomed to things such as Single Page Applications or at least applications that have a consistent look and feel. Depending on the use case, a user may feel uncomfortable switching to the FusionAuth front end to manage users. While FusionAuth has a well designed and functional front end, the look and feel will likely be different than your application. By using the APIs you can still use all of your own style sheets and design principles.
+
+You may ask yourself, “If I create a front end for user management in my application, shouldn’t I just create the authentication and authorization framework as well?” The answer to that question is a resounding “No!” The security, power and flexibility FusionAuth gives you has taken a long time to create and perfect. You should focus on providing the unique business value your application provides versus reinventing the authentication wheel. The best part is that you can ride on the shoulders of giants by creating a simple user management interface around the security and power of FusionAuth. You get the best of both worlds. You don’t even have to tell your users you are using FusionAuth. It’s Okay, we don’t mind. We hope we make you look good.
+
+{% include _image.liquid src="/assets/img/blogs/get-more-value/one-does-not-simply.png" alt="One does not simply implement user access management" class="img-fluid" figure=false %}
+
+<br>
+
```

Comment:
Might be nice to mention the alternative:

Use a custom admin form, https://fusionauth.io/docs/v1/tech/core-concepts/tenants#form-settings but that may not be flexible enough (in terms of validation, integration or business logic), may cost too much, and/or you may want to limit access to the admin ui.

Reply:
Added a bit in the Conclusion.